mpt/impl/smtp/cassandra-rabbitmq-object-storage/src/test/java/org/apache/james/mpt/smtp/CassandraRabbitMQAwsS3SmtpTestRuleFactory.java

```diff
@@ -54,6 +54,7 @@ private static GuiceJamesServer createJamesServer(Host cassandraHost, DockerAwsS
         return GuiceJamesServer.forConfiguration(configuration)
             .combineWith(CassandraJamesServerMain.CASSANDRA_SERVER_CORE_MODULE,
                 SmtpTestRule.SMTP_PROTOCOL_MODULE,
+                CassandraJamesServerMain.CASSANDRA_EVENT_STORE_JSON_SERIALIZATION_DEFAULT_MODULE,
```

Comment:
Why isn't this part of CASSANDRA_SERVER_CORE_MODULE ?

Reply:
Makers sense actually, let me change that

Reply:
If I remember correctly, it would not work because it's linked to the MQ (and there are several of them).

Reply:
FYI: It did work when I moved it to `CASSANDRA_SERVER_CORE_MODULE`